packages/storybook-html/src/components/core/colour-apply.story.mdx

```diff
@@ -177,9 +177,15 @@ Backgrounds are used to differentiate parts of an interface, and to create conta
 
 See for example: Sections, Cards, Table -- *add links eventually*
 
+The SASS function 'mix' is used to create variants for UQ neutral tones only. In all other UQ primary or secondary colours, the SASS function 'lighten' or 'darken' is used to create it's variants.
```

Comment:
The audience doesn't need to know these details and it distracts from the colour usage guidance.

Reply:
Removed